core/register.go

```diff
@@ -38,6 +38,7 @@ import (
 	clabnodesvr_cat9kv "github.com/srl-labs/containerlab/nodes/vr_cat9kv"
 	clabnodesvr_csr "github.com/srl-labs/containerlab/nodes/vr_csr"
 	clabnodesvr_freebsd "github.com/srl-labs/containerlab/nodes/vr_freebsd"
+	clabnodescisco_vios "github.com/srl-labs/containerlab/nodes/cisco_vios"
```

Comment:
there is also `index.md`, `readme.md` and `vrnetlab.md` files where the links to the new systems should be added